app/views/admin/badges/edit.html.erb

```diff
@@ -14,8 +14,8 @@
     </div>
 
     <div class="form-group">
-      <%= form.label :slug, "Slug:" %>
-      <%= form.text_field :slug, class: "form-control" %>
+      <%= form.label :slug, "Slug - auto generated based on the title." %>
```

Comment:
```suggestion
      <%= form.label :slug, "Slug (auto-generated based on the title)" %>
```
But, to your point in the other comment...why keep this here at all if it can't be edited and causes confusion? 🤔 

We could perhaps change it to _not_ be a text field at all (maybe it could render next to the title field above?). The fact that this is a field but is _always_ disabled sort of feels like an anti-pattern to me 🤷🏽

Reply:
Yeah... it's a fair point. I guess I wonder if the slug needs to be visible somewhere, but maybe not? We could probably just display the full path somewhere on the form.

Reply:
Okay, I removed it and added a description in the title :)

![image](https://user-images.githubusercontent.com/17884966/117059749-1fc30300-acee-11eb-945f-2fe67b876629.png)